package.json

```diff
@@ -38,13 +40,18 @@
     "eslint-plugin-prettier": "^5.0.1",
     "eslint-plugin-react": "^7.33.2",
     "husky": "^8.0.3",
+    "identity-obj-proxy": "^3.0.0",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
     "lint-staged": "^15.0.2",
     "prettier": "^3.0.3",
     "react-scripts": "^5.0.1",
     "stylelint": "^14.16.1",
     "stylelint-config-prettier": "^9.0.5",
     "stylelint-config-standard": "^29.0.0",
     "stylelint-order": "^6.0.3",
+    "ts-jest": "^29.1.1",
     "typescript": "^4.9.5"
-  }
+  },
+  "type": "module"
```

Comment:
Not sure why you still use the `type` attribute set to **module**.

Reply:
Fixed in the [last commit](https://github.com/zuko4celik/movies-app-new/pull/24/commits/2c9a79b44c2ed1a1bfa637b9d8af0a4b3fe064be).